spotbugs/src/main/java/edu/umd/cs/findbugs/ba/obl/ObligationAnalysis.java

```diff
@@ -89,6 +90,8 @@
 
     static final ClassDescriptor willClose = DescriptorFactory.createClassDescriptor(WillClose.class);
 
+    private static final org.slf4j.Logger LOG = LoggerFactory.getLogger(ObligationAnalysis.class);
```

Comment:
We're already importing `LoggerFactory`, may as well import `Logger` too. And this could use the `MethodHandles.lookup().lookupClass()` approach to be copy-paste friendly.

Reply:
Fixed. Thanks :)